### PR TITLE
formatting fix according to IDE0055

### PR DESCRIPTION
**Pull request by SimaTian**
### Context
I've had some issues with code review formatting suggestions. Most of those were avoidable via rule IDE0055 which is only a suggestion. This is a first rounds of changes to then hopefully make IDE0055.severity = warning toggle possible.

### Changes Made
Formatting fixes.

### Testing
No functionality has changed. Only formatting. Our current tests should still apply.

### Notes
First commit is via hand, second via VS cleanup feature that I found later.

### CODE REVIEW

File: src/Build/BuildCheck/Infrastructure/CheckWrapper.cs [hunk 1]
```diff
@@ -131,7 +131,7 @@ private void AddDiagnostic(CheckConfigurationEffective configurationEffective)
 
         switch (configurationEffective.Severity)
         {
-            
+
```
Comment (rainersigwald) [inline src/Build/BuildCheck/Infrastructure/CheckWrapper.cs:134]: I'd actually just fully delete this one
```suggestion
```

File: src/Build/Definition/Project.cs [hunk 2]
```diff
@@ -2763,7 +2763,7 @@ private static IEnumerable<ProjectItemElement> GetItemElementsThatMightAffectIte
                         itemElement.RemoveLocation == null);
 
                 // add the include operation that created the project item element
-                return [item.Xml, ..relevantElementsAfterInclude];
+                return [item.Xml, .. relevantElementsAfterInclude];
```
Comment (rainersigwald) [inline src/Build/Definition/Project.cs:2766]: Is this really the canonical style? I definitely expected the way it was before.

File: src/BuildCheck.UnitTests/EditorConfig_Tests.cs [hunk 3]
```diff
@@ -1038,7 +1038,7 @@ public void LongLines()
             properties);
     }
 
-    
+
```
Comment (rainersigwald) [inline src/BuildCheck.UnitTests/EditorConfig_Tests.cs:1041]: Is there a separate rule for "only ever one blank line"? I'd just delete these . . . but not a big deal, this PR is a strict improvement.

**Review by rainersigwald — APPROVED** [review]

File: src/Build/Definition/Project.cs (hunk 2 above)
Comment (SimaTian) [reply src/Build/Definition/Project.cs:2766]: It surprised me too, but yes, this is what the rule requested. I recall that I've done this one manually at least on one occasion. Took me a while to figure out what it wanted.
Code modified with this PR passes through build.cmd without warnings so it should be correct - as least as far as the .editorconfig is concerned.

File: src/BuildCheck.UnitTests/EditorConfig_Tests.cs (hunk 3 above)
Comment (SimaTian) [reply src/BuildCheck.UnitTests/EditorConfig_Tests.cs:1041]: [doesn't seem so, here is the list.](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/csharp-formatting-options#new-line-options)
interesting and kind of surprising.

Comment (rainersigwald) [reply src/BuildCheck.UnitTests/EditorConfig_Tests.cs:1041]: Looks like there's `dotnet_style_allow_multiple_blank_lines_experimental` but no concrete plans to make it non-`experimental`: https://github.com/dotnet/roslyn/issues/60539.

File: src/Build.UnitTests/BackEnd/BinaryTranslator_Tests.cs [hunk 4]
```diff
@@ -258,7 +258,7 @@ public static IEnumerable<object[]> GetBuildExceptionsAsTestData()
         [MemberData(nameof(GetBuildExceptionsAsTestData))]
         public void TestSerializationOfBuildExceptions(Type exceptionType)
         {
-            Exception e = (Exception)Activator.CreateInstance(exceptionType, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.CreateInstance | BindingFlags.Instance, null, new object[]{"msg", new GenericBuildTransferredException() }, System.Globalization.CultureInfo.CurrentCulture);
+            Exception e = (Exception)Activator.CreateInstance(exceptionType, BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.CreateInstance | BindingFlags.Instance, null, new object[] { "msg", new GenericBuildTransferredException() }, System.Globalization.CultureInfo.CurrentCulture);
```
Comment (maridematte) [inline src/Build.UnitTests/BackEnd/BinaryTranslator_Tests.cs:261]:
```suggestion
            Exception e = (Exception)Activator.CreateInstance(
            exceptionType, 
            BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.CreateInstance | BindingFlags.Instance,
            null, 
            new object[] { "msg", new GenericBuildTransferredException() },
            System.Globalization.CultureInfo.CurrentCulture);
```

File: src/Build.UnitTests/BackEnd/TargetBuilder_Tests.cs [hunk 5]
```diff
@@ -771,7 +771,7 @@ public void TestAfterTargetsEmpty()
             TargetBuilder builder = (TargetBuilder)_host.GetComponent(BuildComponentType.TargetBuilder);
             IConfigCache cache = (IConfigCache)_host.GetComponent(BuildComponentType.ConfigCache);
             (string name, TargetBuiltReason reason)[] target = { ("Build", TargetBuiltReason.None) }
-;            BuildRequestEntry entry = new BuildRequestEntry(CreateNewBuildRequest(1, target), cache[1]);
+; BuildRequestEntry entry = new BuildRequestEntry(CreateNewBuildRequest(1, target), cache[1]);
```
Comment (maridematte) [inline src/Build.UnitTests/BackEnd/TargetBuilder_Tests.cs:774]: Can we just put the semicolon on the above line?

**Review by maridematte — APPROVED** [review]
Looks good to me. Just a few nits since the code in those is already being touched, but feel free to ignore them.

File: src/Build.UnitTests/BackEnd/BinaryTranslator_Tests.cs (hunk 4 above)
Comment (SimaTian) [reply src/Build.UnitTests/BackEnd/BinaryTranslator_Tests.cs]: Is it ok if I do one more indent there to make the createInstance parameters more pronounced?:
```
Exception e = (Exception)Activator.CreateInstance(
    exceptionType,
    BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.CreateInstance | BindingFlags.Instance,
    null,
    new object[] { "msg", new GenericBuildTransferredException() },
    System.Globalization.CultureInfo.CurrentCulture);
```

Comment (maridematte) [reply src/Build.UnitTests/BackEnd/BinaryTranslator_Tests.cs]: Yeah, I think that looks better.

**Review by JanKrivanek — APPROVED** [review]
Thank you!

**Pull request merged**